src/landscape_type.h

```diff
@@ -30,6 +30,7 @@ enum Borders {
 	BORDER_SE = 1,
 	BORDER_SW = 2,
 	BORDER_NW = 3,
+	BORDERS_ALL = 15,
```

Comment:
What feels a bit confusing, that the first 4 are bits, and this one is not. Maybe setting it to `(1 << BORDER_NE) | (1 << BORDE_SE) | ...` makes it a bit more clear what the relation is between these two. As it isn't random, the value 15. 14 won't work, if I am reading the code right.

Reply:
#13290 makes this nicer.